### DISCUSSION TITLE
Replay Render in all 4 modes: Transform Your osu! Gameplay into Stunning MP4 Videos

### DISCUSSION THREAD

**Discussion by LilSanik** [ppy/osu#23473] [Ideas] [answered]
**Proposal:**
We propose the development of a replay render feature that will allow players to export their osu! replays into high-quality MP4 videos. This feature will enable players to showcase their gameplay on various platforms such as social media, YouTube, and Twitch, in a more professional and visually appealing way.

**Functionality:**
The replay render feature would essentially consist of a button or section that allows users to export their replays in MP4 format, giving them the ability to choose the desired quality and resolution.

**Benefits:**
The replay render feature will provide several benefits to osu! players, including:

- Low-end PC users will be able to share their gameplay content on social media platforms.
- The ability to create high-quality showcases of their gameplay that can be shared with the community.

**Conclusion:**
We believe that the replay render feature will be a valuable addition to the osu! game and community. We look forward to the possibility of implementing this feature and seeing the creative and unique videos that osu! players will create.

**Comment by monochrome22** [comment, marked as answer]
It was talked about at https://github.com/ppy/osu/discussions/12986 and https://github.com/ppy/osu/discussions/21729
